.gitignore

```diff
@@ -7,6 +7,7 @@ dist/*
 
 node_modules/
 tmp/
+*.war
```

Comment:
I'd rather remove it in a separate PR once the main Marathon build is using the `war`, just in case. Is that too cautious?